rustls/Cargo.toml

```diff
@@ -40,6 +40,7 @@ aws_lc_rs = ["dep:aws-lc-rs", "webpki/aws_lc_rs"]
 aws-lc-rs = ["aws_lc_rs"] # Alias because Cargo features commonly use `-`
 brotli = ["dep:brotli", "dep:brotli-decompressor", "std"]
 ring = ["dep:ring", "webpki/ring"]
+custom_provider = []
```

Comment:
I think we wanted to standardize on dashes over underscores in feature names now?

Reply:
https://github.com/rustls/rustls/issues/1961 seems to go in that direction but don't mention the `read_buf` feature so I was confused

Reply:
Probably still use a dash here.

Reply:
I've changed it to a dash in the latest push ;)